docs/guide/docs/tutorials/hitl.md

```diff
@@ -52,3 +54,155 @@ The agent can then navigate to the appropriate conversation and take over the co
 Once the agent is done communicating with the user, they can unpause the conversation.
 
 It is also possible for the user to unpause the conversation by calling the API endpoint.
+
+## HITL-next (beta)
+
+This revamped HITL works on **all existing and future channels**. It supports all features of its predecessor and a few more :
+
+- Multi-agents ( _enterprise edition only_ )
+- Human handoff from workflow x
+- Realtime agent interface
+- Agent notes
+- End user profile
+- Configurable transfer and assignment messages
+- Message templates
+- Basic queuing and assignent system
+- Basic labeling
+- Documented HTTP API (_coming soon_)
+- Webhook integration (_coming soon_)
+
+### Requirements
+
+This module uses the `channel-web` to display conversations, so make sure it's enabled. Moreover, a multi-agents setup needs an enterprise license.
+
+### Setup
+
+Simply turn on HITL-next on the module management page of you Botpress admin.
+[enable-module image here]
```

Comment:
You can link to https://botpress.com/docs/main/module#enabling-or-disabling-modules

Reply:
fixed